src/ads/ads-controller.js

```diff
@@ -94,19 +94,23 @@ class AdsController extends FakeEventTarget implements IAdsController {
 
   /**
    * Play an ad on demand.
-   * @param {string} adTagUrl - The ad tag url to play.
+   * @param {PKAdPod} adPod - The ad pod play.
    * @instance
    * @memberof AdsController
    * @returns {void}
    */
-  playAdNow(adTagUrl: string): void {
+  playAdNow(adPod: PKAdPod): void {
     if (this.isAdBreak()) {
       AdsController._logger.warn('Tried to call playAdNow during an adBreak');
     } else {
+      this._player.pause();
```

Comment:
check if needed, IMA should fire CONTENT_PAUSE_REQUESTED and then we pause the content

Reply:
seems related to below request for play so this might be a bug - let's understand this